<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddCarWindow</class>
 <widget class="QDialog" name="AddCarWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>408</width>
    <height>345</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add car</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QLineEdit" name="lineEditYear">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>121</width>
     <height>24</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Year</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditRegDate">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>121</width>
     <height>24</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Registration Date</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditModel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>121</width>
     <height>24</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Model</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditColor">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>121</width>
     <height>24</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Color</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditBrand">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>121</width>
     <height>24</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Brand</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPrice">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>121</width>
     <height>24</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Price</string>
   </property>
  </widget>
  <widget class="QLabel" name="logoHandler">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>50</y>
     <width>231</width>
     <height>191</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>331</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="styleSheet">
       <string notr="true">QPushButton
{
background-color: rgb(245, 245, 245);
border:none;
color: rgb(2, 2, 2);
border-left: 1px solid rgb(245, 245, 245);
border-right: 1px solid rgb(245, 245, 245);
border-bottom: 5px solid rgb(245, 245, 245);
}
QPushButton:hover {
    color: #000000;
    background-color: rgb(200, 200, 200); 
    border: 0px solid rgb(0, 0, 0);
}</string>
      </property>
      <property name="text">
       <string>Add photo</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonCreate">
      <property name="styleSheet">
       <string notr="true">QPushButton
{
background-color: rgb(245, 245, 245);
border:none;
color: rgb(2, 2, 2);
border-left: 1px solid rgb(245, 245, 245);
border-right: 1px solid rgb(245, 245, 245);
border-bottom: 5px solid rgb(245, 245, 245);
}
QPushButton:hover {
    color: #000000;
    background-color: rgb(200, 200, 200); 
    border: 0px solid rgb(0, 0, 0);
}</string>
      </property>
      <property name="text">
       <string>Create</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="lineEditDate">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Date</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEditBrand</tabstop>
  <tabstop>lineEditModel</tabstop>
  <tabstop>lineEditYear</tabstop>
  <tabstop>lineEditColor</tabstop>
  <tabstop>lineEditPrice</tabstop>
  <tabstop>lineEditRegDate</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
